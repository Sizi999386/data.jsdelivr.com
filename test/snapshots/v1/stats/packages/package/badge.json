{
	"/v1/stats/packages/npm/package-2/badge": "<svg xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" width=\"146\" height=\"20\" role=\"img\" aria-label=\"jsDelivr: 22k hits/month\"><title>jsDelivr: 22k hits/month</title><g shape-rendering=\"crispEdges\"><rect width=\"51\" height=\"20\" fill=\"#555\"/><rect x=\"51\" width=\"95\" height=\"20\" fill=\"#ff5627\"/></g><g fill=\"#fff\" text-anchor=\"middle\" font-family=\"Verdana,Geneva,DejaVu Sans,sans-serif\" text-rendering=\"geometricPrecision\" font-size=\"110\"><text x=\"265\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"410\">jsDelivr</text><text x=\"975\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"850\">22k hits/month</text></g></svg>",
	"/v1/stats/packages/npm/package-2/badge?type=rank": "<svg xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" width=\"120\" height=\"20\" role=\"img\" aria-label=\"jsDelivr rank: #117\"><title>jsDelivr rank: #117</title><g shape-rendering=\"crispEdges\"><rect width=\"81\" height=\"20\" fill=\"#555\"/><rect x=\"81\" width=\"39\" height=\"20\" fill=\"#ff5627\"/></g><g fill=\"#fff\" text-anchor=\"middle\" font-family=\"Verdana,Geneva,DejaVu Sans,sans-serif\" text-rendering=\"geometricPrecision\" font-size=\"110\"><text x=\"415\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"710\">jsDelivr rank</text><text x=\"995\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"290\">#117</text></g></svg>",
	"/v1/stats/packages/npm/package-2/badge?type=type-rank": "<svg xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" width=\"142\" height=\"20\" role=\"img\" aria-label=\"jsDelivr npm rank: #57\"><title>jsDelivr npm rank: #57</title><g shape-rendering=\"crispEdges\"><rect width=\"109\" height=\"20\" fill=\"#555\"/><rect x=\"109\" width=\"33\" height=\"20\" fill=\"#ff5627\"/></g><g fill=\"#fff\" text-anchor=\"middle\" font-family=\"Verdana,Geneva,DejaVu Sans,sans-serif\" text-rendering=\"geometricPrecision\" font-size=\"110\"><text x=\"555\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"990\">jsDelivr npm rank</text><text x=\"1245\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"230\">#57</text></g></svg>",
	"/v1/stats/packages/npm/package-2/badge?type=type-rank&period=s-year": {
		"date": "2010-12-31",
		"status": 400,
		"message": "Invalid parameter value: `period` must be one of [day, week, month, year].",
		"links": {
			"documentation": "http://localhost:4400/docs/data.jsdelivr.com#get-/v1/stats/packages/npm/-package-/badge"
		}
	},
	"/v1/stats/packages/npm/package-2/badge?type=type-rank&period=year": "<svg xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" width=\"142\" height=\"20\" role=\"img\" aria-label=\"jsDelivr npm rank: #57\"><title>jsDelivr npm rank: #57</title><g shape-rendering=\"crispEdges\"><rect width=\"109\" height=\"20\" fill=\"#555\"/><rect x=\"109\" width=\"33\" height=\"20\" fill=\"#ff5627\"/></g><g fill=\"#fff\" text-anchor=\"middle\" font-family=\"Verdana,Geneva,DejaVu Sans,sans-serif\" text-rendering=\"geometricPrecision\" font-size=\"110\"><text x=\"555\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"990\">jsDelivr npm rank</text><text x=\"1245\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"230\">#57</text></g></svg>",
	"/v1/stats/packages/npm/@scope/package-1/badge?style=rounded": "<svg xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" width=\"146\" height=\"20\" role=\"img\" aria-label=\"jsDelivr: 21k hits/month\"><title>jsDelivr: 21k hits/month</title><linearGradient id=\"s\" x2=\"0\" y2=\"100%\"><stop offset=\"0\" stop-color=\"#bbb\" stop-opacity=\".1\"/><stop offset=\"1\" stop-opacity=\".1\"/></linearGradient><clipPath id=\"r\"><rect width=\"146\" height=\"20\" rx=\"3\" fill=\"#fff\"/></clipPath><g clip-path=\"url(#r)\"><rect width=\"51\" height=\"20\" fill=\"#555\"/><rect x=\"51\" width=\"95\" height=\"20\" fill=\"#ff5627\"/><rect width=\"146\" height=\"20\" fill=\"url(#s)\"/></g><g fill=\"#fff\" text-anchor=\"middle\" font-family=\"Verdana,Geneva,DejaVu Sans,sans-serif\" text-rendering=\"geometricPrecision\" font-size=\"110\"><text aria-hidden=\"true\" x=\"265\" y=\"150\" fill=\"#010101\" fill-opacity=\".3\" transform=\"scale(.1)\" textLength=\"410\">jsDelivr</text><text x=\"265\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"410\">jsDelivr</text><text aria-hidden=\"true\" x=\"975\" y=\"150\" fill=\"#010101\" fill-opacity=\".3\" transform=\"scale(.1)\" textLength=\"850\">21k hits/month</text><text x=\"975\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"850\">21k hits/month</text></g></svg>",
	"/v1/stats/packages/npm/@scope/package-1/badge?type=type-rank": "<svg xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" width=\"142\" height=\"20\" role=\"img\" aria-label=\"jsDelivr npm rank: #58\"><title>jsDelivr npm rank: #58</title><g shape-rendering=\"crispEdges\"><rect width=\"109\" height=\"20\" fill=\"#555\"/><rect x=\"109\" width=\"33\" height=\"20\" fill=\"#ff5627\"/></g><g fill=\"#fff\" text-anchor=\"middle\" font-family=\"Verdana,Geneva,DejaVu Sans,sans-serif\" text-rendering=\"geometricPrecision\" font-size=\"110\"><text x=\"555\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"990\">jsDelivr npm rank</text><text x=\"1245\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"230\">#58</text></g></svg>"
}
