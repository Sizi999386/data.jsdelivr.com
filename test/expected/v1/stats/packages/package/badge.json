{
	"/v1/stats/packages/npm/package-2/badge": "<svg xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" width=\"146\" height=\"20\" role=\"img\" aria-label=\"jsDelivr: 22k hits/month\"><title>jsDelivr: 22k hits/month</title><g shape-rendering=\"crispEdges\"><rect width=\"51\" height=\"20\" fill=\"#555\"/><rect x=\"51\" width=\"95\" height=\"20\" fill=\"#ff5627\"/></g><g fill=\"#fff\" text-anchor=\"middle\" font-family=\"Verdana,Geneva,DejaVu Sans,sans-serif\" text-rendering=\"geometricPrecision\" font-size=\"110\"><text x=\"265\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"410\">jsDelivr</text><text x=\"975\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"850\">22k hits/month</text></g></svg>",
	"/v1/stats/packages/npm/package-2/badge?type=rank": "<svg xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" width=\"120\" height=\"20\" role=\"img\" aria-label=\"jsDelivr rank: #117\"><title>jsDelivr rank: #117</title><g shape-rendering=\"crispEdges\"><rect width=\"81\" height=\"20\" fill=\"#555\"/><rect x=\"81\" width=\"39\" height=\"20\" fill=\"#ff5627\"/></g><g fill=\"#fff\" text-anchor=\"middle\" font-family=\"Verdana,Geneva,DejaVu Sans,sans-serif\" text-rendering=\"geometricPrecision\" font-size=\"110\"><text x=\"415\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"710\">jsDelivr rank</text><text x=\"995\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"290\">#117</text></g></svg>",
	"/v1/stats/packages/npm/package-2/badge?type=type-rank": "<svg xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" width=\"142\" height=\"20\" role=\"img\" aria-label=\"jsDelivr npm rank: #57\"><title>jsDelivr npm rank: #57</title><g shape-rendering=\"crispEdges\"><rect width=\"109\" height=\"20\" fill=\"#555\"/><rect x=\"109\" width=\"33\" height=\"20\" fill=\"#ff5627\"/></g><g fill=\"#fff\" text-anchor=\"middle\" font-family=\"Verdana,Geneva,DejaVu Sans,sans-serif\" text-rendering=\"geometricPrecision\" font-size=\"110\"><text x=\"555\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"990\">jsDelivr npm rank</text><text x=\"1245\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"230\">#57</text></g></svg>",
	"/v1/stats/packages/npm/package-2/badge?type=type-rank&period=s-year": {
		"date": "2010-12-31",
		"status": 400,
		"message": "Invalid parameter value: `period` must be one of [day, week, month, year].",
		"links": {
			"documentation": "https://www-jsdelivr-com-pr-400.onrender.com/docs/data.jsdelivr.com#get-/v1/stats/packages/npm/-package-/badge"
		}
	},
	"/v1/stats/packages/npm/package-2/badge?type=type-rank&period=year": "<svg xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" width=\"142\" height=\"20\" role=\"img\" aria-label=\"jsDelivr npm rank: #57\"><title>jsDelivr npm rank: #57</title><g shape-rendering=\"crispEdges\"><rect width=\"109\" height=\"20\" fill=\"#555\"/><rect x=\"109\" width=\"33\" height=\"20\" fill=\"#ff5627\"/></g><g fill=\"#fff\" text-anchor=\"middle\" font-family=\"Verdana,Geneva,DejaVu Sans,sans-serif\" text-rendering=\"geometricPrecision\" font-size=\"110\"><text x=\"555\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"990\">jsDelivr npm rank</text><text x=\"1245\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"230\">#57</text></g></svg>",
	"/v1/stats/packages/npm/@scope/package-1/badge?type=type-rank": "<svg xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" width=\"142\" height=\"20\" role=\"img\" aria-label=\"jsDelivr npm rank: #58\"><title>jsDelivr npm rank: #58</title><g shape-rendering=\"crispEdges\"><rect width=\"109\" height=\"20\" fill=\"#555\"/><rect x=\"109\" width=\"33\" height=\"20\" fill=\"#ff5627\"/></g><g fill=\"#fff\" text-anchor=\"middle\" font-family=\"Verdana,Geneva,DejaVu Sans,sans-serif\" text-rendering=\"geometricPrecision\" font-size=\"110\"><text x=\"555\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"990\">jsDelivr npm rank</text><text x=\"1245\" y=\"140\" transform=\"scale(.1)\" fill=\"#fff\" textLength=\"230\">#58</text></g></svg>"
}
